package.json

```diff
@@ -89,11 +90,12 @@
     "btoa": "^1.1.2",
     "bunyan": "^2.0.2",
     "chokidar": "^3.2.2",
-    "cozy-client": "^11.2.1",
+    "cozy-client": "^14.6.0",
     "cozy-client-js": "^0.17.1",
+    "cozy-stack-client": "^14.1.2",
```

Comment:
no need to add it since you import the FetchError from cozy-client

Reply:
I need it for the linter :/

Reply:
OMG `const { FetchError } = require('cozy-stack-client')` my bad